### PR TITLE
Deprecate finance wiki

### PR DESCRIPTION
**Pull request by jdswinbank**
Addresses #224.

### CODE REVIEW

**Comment by jdswinbank** [comment]
I'd like to request review from @aaryapatil but that doesn't seem to be possible — I guess she needs to be given permission. Please you could take care of this when you review, Erik or Kelle?

**Comment by hamogu** [comment]
Added @aaryapatil to finance-commitee role (which has write access in this repro) and requested her review here.

**Review by kelle — APPROVED** [review]

**Comment by jdswinbank** [comment]
Following discussion in the meeting, we should wait for #231 to merge, rebase this, then fix any remaining use of “interim”.

**Review by kelle — APPROVED** [review]

**Pull request merged**
